docs/comparisons/README.md

```diff
@@ -0,0 +1,9 @@
+# Comparisons between .NET Standard and other platforms
+
+Many have asked for a diff between .NET Standard and other existing platforms.
```

Comment:
I'll add a description. The way to read the diffs is like this:

X vs. Y
+ Didn't exist in X, but exist in Y
- Did exist in X but doesn't exist in Y